### ISSUE TITLE
Change default category "Blog" title ?

### ISSUE THREAD

**Issue by kevin5396** [kelvinh/org-page#202]
Can I use another name like "Post" as the title for that category?

**Comment by kelvinh** [comment]
Sorry, "blog" is a default category cannot be removed, the uri cannot be changed, however, you can define a `:label` in `op/category-config-alist` to change the name shown on pages. See function `op/get-category-name` and variable `op/category-config-alist` for detail.

**Comment by kelvinh** [comment]
Try add the following option in your org file:

`#+URI: /the/url/you/wanted/here`

**Comment by nasseralkmim** [comment]
the url of individual posts are correct. But the url in the navigation bar is using the category name (blog) instead of the label (notes).

I think it is related with the variable: category-uri. Is it getting the name 'blog' or the assigned label?

**Comment by kelvinh** [comment]
Yes, it's rendered by `category-uri`, only the shown name is controlled by assigned label, the uri is not.

**Comment by kelvinh** [comment]
The newest commit has fixed this problem, happy hacking!

**Comment by nasseralkmim** [comment]
That's great, thank you.

**Comment by kelvinh** [comment]
What is broken? the ignore list cannot ignore the blog category, as it's one of the default categories.

**Comment by nasseralkmim** [comment]
It seems that a category with a different label produce a new folder with the label name without a index.html, and since the url is generated by a folder with a index.html you don't get the list of entries in the category.

It's better to keep the way it was, the label only change the name displayed, not the url.

**Comment by kelvinh** [comment]
Yes, you are right, the commit has been reverted. There is another dirty way to achieve this: edit the template **nav.mustache**, put the things you wanted there directly.
